Docker build image Go 1.17 dependency error

I get an error while building the docker image

```
docker build -t wot-discovery-testing .

# golang.org/x/crypto/acme/autocert/internal/acmetest
/go/pkg/mod/golang.org/x/crypto@v0.0.0-20220427172511-eb4f295cb31f/acme/autocert/internal/acmetest/ca.go:748:13: undefined: io.ReadAll
note: module requires Go 1.17
# golang.org/x/sys/execabs
/go/pkg/mod/golang.org/x/sys@v0.0.0-20220502124256-b6088ccd6cba/execabs/execabs_go119.go:11:6: isGo119ErrDot redeclared in this block
	previous declaration at /go/pkg/mod/golang.org/x/sys@v0.0.0-20220502124256-b6088ccd6cba/execabs/execabs_go118.go:9:31
note: module requires Go 1.17
The command '/bin/sh -c go get .../.' returned a non-zero code: 2
```

To make it work I had to change `RUN go get .../.` in the Dockerfile into `RUN go install`, I can make a PR for that if need be.

--- a/directory/Dockerfile
+++ b/directory/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.15-alpine
+FROM golang:1.17-alpine
 
 COPY . /home
 
@@ -7,6 +7,6 @@ WORKDIR /home
 ENV CGO_ENABLED=0
 
 # download dependencies
-RUN go get .../.
+RUN go mod download
 
 ENTRYPOINT ["go", "test"]

